specs/phase0/fork-choice.md

```diff
@@ -182,7 +182,10 @@ def get_latest_attesting_balance(store: Store, root: Root) -> Gwei:
             and get_ancestor(store, store.latest_messages[i].root, store.blocks[root].slot) == root)
     ))
     proposer_score = Gwei(0)
-    if store.proposer_boost_root != Root() and root == store.proposer_boost_root:
+    if (
+        store.proposer_boost_root != Root()
+        and get_ancestor(store, store.proposer_boost_root, store.blocks[root].slot) == root
```

Comment:
generally should avoid multi-line ifs. This is also a bit secretly too pythonic (it assumes that if the first bool fails, the second is not executed, avoiding the undefined behavior if `Root()` being passed into `get_ancestor`

suggest  somthing like the following
```python
    if store.proposer_boost_root != Root():
        if get_ancestor():
            ...
```

An alternative, I personally prefer this:

```python
    if store.proposer_boost_root == Root():
        return attestation_score
    proposer_score = Gwei(0)
    if get_ancestor():
        ...
    return attestation_score + proposer_score
```

I think that very nicely reads as "If proposer_boost is not enabled, just return the attestation_score, otherwise calculate the proposer score and return the sum"